### PR TITLE
Format code using 'cargo fmt'

### PR DESCRIPTION
**Pull request by Atul9**

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+max_width = 100
```
Comment (sfackler) [inline rustfmt.toml]: Just running with defaults seems fine.

Comment (Atul9) [reply rustfmt.toml]: The rustfmt.toml is generated using rustfmt with default settings.

Comment (sfackler) [reply rustfmt.toml]: So then this file doesn't need to exist, right?

Comment (Atul9) [reply rustfmt.toml]: while contributing to other rust opensource projects I got to know that - `without an included rustfmt.toml file runs the risk fo some things being formatted randomly at random times` reference - https://github.com/sagebind/sluice/pull/4#issuecomment-536084517, hence I included it.

Comment (sfackler) [reply rustfmt.toml]: I'm not convinced that random format changes will happen.

Comment (Atul9) [reply rustfmt.toml]: I am ok with not including rustfmt.toml in this PR, should I remove it?

Comment (sfackler) [reply rustfmt.toml]: Yes

Comment (Atul9) [reply rustfmt.toml]: done, I have updated the PR.

**Pull request merged**